tests/benchmarking/TatumQuantizationFitBenchmarking.cpp

```diff
@@ -144,8 +144,9 @@ TEST_CASE("TatumQuantizationFitBenchmarking")
    // Verify that the input has not changed between this run and the previous
    // one, or peformance of the algorithm can't be compared either.
    constexpr auto previousChecksum = -177205.328125f;
+   // TODO: FIX THE TOLERANCE
    REQUIRE(
-      ValueIsUnchanged(outputDir / "checksum.txt", previousChecksum, checksum));
+      ValueIsUnchanged(outputDir / "checksum.txt", previousChecksum, checksum, 1.0f));
```

Comment:
1.0f, so much ? At the scale of 177205 that's probably negligible, yeah, ok.  How about platform-dependent references ? Unreasonably much work ?
What TODO do you have in mind?

Reply:
It's to make sure that the set of input files hasn't changed before we do the benchmarking, and that the same evaluation is indeed being reproduced.
I'll talk about it tomorrow in the talk :D
> Is it possible not to use FP at all and use a more stable checksum instead?

I'm open, but for now I think that will do.